packages/editor/src/components/panel-color-settings/index.js

```diff
@@ -17,58 +17,107 @@ import ColorPaletteControl from '../color-palette/control';
 import withColorContext from '../color-palette/with-color-context';
 import { getColorObjectByColorValue } from '../colors';
 
+const customColorsDisableForSetting = ( disableCustomColors, colorSetting ) => {
```

Comment:
nitpick: the name of this function could start with 'has' like the other two functions, and should probably use the adjective 'disabled' instead of the verb 'disable'.